content/en/concepts/payment-methods.md

```diff
@@ -5,15 +5,32 @@ position: 10
 category: Payment Concepts
 ---
 
-Conteúdo
+Each payment method is defined by 3 key characteristics:
+- **Is it pull or push**: How the funds for the customer are acquired from the customer.
+- **Flow**: The type of action the user takes to make the payment.
```

Comment:
After reading everything, the Flow is never mentioned again and I don't understand why this reference is necessary.

Reply:
Removed